### ISSUE TITLE
wrong classification of AD as a rare disease

### ISSUE THREAD

**Issue by ElaineMcA** [opentargets/platform#104]
@deniseOme commented on [Thu Sep 07 2017](https://github.com/opentargets/rest_api/issues/101)

Is Alzheimer's a rare disease @elipapa @gkos-bio @apierleoni @ckongEbi @iandunham ?

https://www.targetvalidation.org/evidence/ENSG00000080815/EFO_0000249

![screen shot 2017-09-07 at 14 14 39](https://user-images.githubusercontent.com/6472381/30165123-ceacd790-93d6-11e7-9568-2f31cd2f8bb7.png)

---

@elipapa commented on [Mon Nov 13 2017](https://github.com/opentargets/rest_api/issues/101#issuecomment-343894496)

In this case the mapping is not about the disease. We call it "rare disease" because the evidence comes from clinvar. 

@iandunham should we relabel it "mendelian" or "curated" ? that would take care of the confusion.

---

@elipapa commented on [Mon Nov 13 2017](https://github.com/opentargets/rest_api/issues/101#issuecomment-343894553)

not an API issue anyways.

---

@emepyc commented on [Mon Nov 13 2017](https://github.com/opentargets/rest_api/issues/101#issuecomment-343895051)

Our distinction between rare and common is just based on the source.
This is our current mappings:
https://github.com/opentargets/webapp/blob/master/app/config/general/default.json#L71

---

@ElaineMcA commented on [Tue Aug 28 2018](https://github.com/opentargets/rest_api/issues/101#issuecomment-416553090)

@gkos-bio What should the action be here please?

**Comment by iandunham** [comment]
I think thisis about mapping of the mutations. At elaset one of the references that comes with  the "Alzheimers disease" mutations from clinvar is about early onset familial (i.e. rare) disease https://europepmc.org/abstract/MED/20301414. That paper says  PSEN1 is responsible for 30%-70% of Early Onset Familial AD. The action here should be to ask EVA/clinvar whether they should actually be classifying these mutations under the familial AD terms.

Does PSEN1 have GWAS for common AD? We don't have any GWAS evidence. 

We can consider renaming the category, but only if that helps users in general - the original decision was based on many users not understanding what "Mendelian" meant. "Curated" isn't the right word. We shouldn't rename the category just based on this case - better to fix the data - if necessary we can arrange for EVA to have a distinction of the data types if they start getting common variants, like we do for somatic mutations.

**Comment by ElaineMcA** [comment]
Have raised with EVA - Need to revisit in the context of EFO 3.  Marking for 18.12.

**Comment by afaulconbridge** [comment]
As this is a data team issue, not a backend or ontology content issue, I'm reassigning it as appropriate

**Comment by AsierGonzalez** [comment]
Did we ever hear back from EVA @ElaineMcA?

**Comment by ElaineMcA** [comment]
@AsierGonzalez  - have asked again for curation feedback and copied you in the note to Cristina.

**Comment by AsierGonzalez** [comment]
After discussion with the EVA team it seems that the issue in this case is the incorrect mapping of the ClinVar condition `Alzheimer disease, type 3 (AD)` to `alzheimer's disease` (EFO_0000249) instead of `earl-onset autosomal dominant Alzhemer disease` (Orphanet_1020).
The objective is to correct it for the next release.

**Comment by AsierGonzalez** [comment]
This is still not resolved in the file received for 19.09

**Comment by AsierGonzalez** [comment]
This is still unresolved. I will ask EVA about it.

**Comment by AsierGonzalez** [comment]
It seems that this issue affects in total 69 evidence strings annotated with `Alzheimer's disease` ([EFO_0000249](http://www.ebi.ac.uk/efo/EFO_0000249)) in the evidence strings when in ClinVar they are linked to `alzheimer disease, type 1`, `alzheimer disease, type 3` or `alzheimer disease, type 4`. All of them should be annotated with `Early-onset autosomal dominant Alzheimer disease` ([Orphanet_1020](http://www.orpha.net/ORDO/Orphanet_1020)) instead.

Issue has been followed up with EVA and I am awaiting feedback to see if it will be fixed for 20.04

**Comment by AsierGonzalez** [comment]
EVA have corrected the EFO mappings of the AD subtypes mentioned above.

However, there are still 20 evidence strings annotated with `EFO_0000249` because the reported condition in ClinVar is `Alzheimer disease (AD)`. Based on the synonyms, `Presenile and senile dementia; Alzheimer's disease; ALZHEIMER DISEASE, FAMILIAL, 1, AUTOSOMAL RECESSIVE`, these should also be annotated with `Orphanet_1020`. Fixing these will be more complex as it is not an ontology mapping issue but the condition is incorrect in the source (ClinVar).

In addition, in the latest submission there is also a variant ([RCV000019448](https://www.ncbi.nlm.nih.gov/clinvar/RCV000019448/)) associated with AD whose reported condition is `alzheimer disease 2`. This is a type of late-onset alzheimer's disease and as such, should me mapped to [EFO_1001870](http://www.ebi.ac.uk/efo/EFO_1001870).

**Comment by AsierGonzalez** [comment]
For the 20.04 release the mappings mentioned above have been manually fixed and there are now no evidence strings annotated with `EFO_0000249`.

A more robust fix for this needs to be discussed with Kirill. The suggested approach would be to map the ClinVar conditions using a two-step approach:

1. Cross-reference the orphanet or OMIM in ClinVar to EFO **if it exists**
2. If not use Zooma to map the condition name

**Caveat:** this approach will give incorrect results if the ClinVar OMIM or Orphanet ids are incorrect. For example, the orphanet id linked to `alzheimer disease 2` in the `RCV000019448` variant mentioned above would map it to `Early-onset autosomal dominant Alzheimer disease` [(Orphanet_1020)](http://www.orpha.net/ORDO/Orphanet_1020) when it should be `late-onset Alzheimers disease` [(Orphanet_1020)](http://www.ebi.ac.uk/efo/EFO_1001870).

**Comment by tskir** [comment]
@AsierGonzalez At this point, we have identified and fixed the following mappings on our side:
* "Alzheimer disease, type {1, 3, 4}" → http://www.orpha.net/ORDO/Orphanet_1020
* "alzheimer disease 2" → http://www.ebi.ac.uk/efo/EFO_1001870

You also listed "Alzheimer disease (AD)" in the spreadsheet of problematic terms, which I wasn't able to find in our records, and so wasn't able to correct. Do you have examples of RCVs with such a trait?

**Comment by AsierGonzalez** [comment]
Thank you for fixing those mappings @tskir. The 20 ClinVar entries that are annotated with `Alzheimer disease (AD)` and are mapped to `EFO_0000249` are the following:
```
RCV000019734
RCV000020306
RCV000020307
RCV000020308
RCV000034924
RCV000687111
RCV000736245
RCV000736246
RCV000736249
RCV000736259
RCV000736261
RCV000736262
RCV000736263
RCV000736264
RCV000736266
RCV000736267
RCV000736269
RCV000815476
RCV000010376
RCV000763144
```
For 20.04 I changed their disease annotation manually in the file you submitted. I can do this again for this release unless you can fix it in your pipeline and submit a new file.

**Comment by AsierGonzalez** [comment]
The mapping of these variants has been corrected manually for 20.06

**Comment by tskir** [comment]
Thanks @AsierGonzaler, I'll take a look at those RCVs (can't promise it will be soon though!)

**Comment by AsierGonzalez** [comment]
@tskir, could you please look into this? The process to manually it is a little cumbersome

**Comment by AsierGonzalez** [comment]
Note to self: this is the jq command to replace AD by EOAD in the evidence file:

`grep "EFO_0000249" cttv012-11-08-2020.json | jq-1.6 -c '.disease.id|="http://www.orpha.net/ORDO/Orphanet_1020" | .disease.name|="Early-onset autosomal dominant Alzheimer disease" | .unique_association_fields.phenotype |= "http://www.orpha.net/ORDO/Orphanet_1020"' > cttv012-11-08-2020_EFO_0000249_subset_corrected.json`

**Comment by andrewhercules** [comment]
Reassigning to 20.11 release to follow up with EVA about mapping variants

**Comment by tskir** [comment]
@AsierGonzalez From now on this should be fixed for all submitted evidence strings, with no intervention required from your side. In fact, if you'd like to take a look, this is already fixed in `cttv012-20200928-test2.json.gz` test evidence string set which I submitted a couple of weeks ago.

From now on, we map the ClinVar term with the main name “Alzheimer disease” and synonyms “Presenile and senile dementia; Alzheimer's disease; ALZHEIMER DISEASE, FAMILIAL, 1, AUTOSOMAL RECESSIVE” to http://www.orpha.net/ORDO/Orphanet_1020, just as we do for types 1, 3, and 4.

I also created a follow-up issue for the manual curation platform to discuss use of context when importing the entities for mapping: https://github.com/EBIvariation/trait-curation/issues/101. In the case of ClinVar, the context constitutes synonyms and existing ontology links.

**Comment by AsierGonzalez** [comment]
That is great @tskir, thank you so much for fixing this. I confirm that  `cttv012-20200928-test2.json.gz` did not have any evidence strings annotated with `EFO_0000249`. I leave the ticket open until you submit the official file for 20.11.

Also, I agree that adding context to the trait mapping tool would be beneficial.
